### PR TITLE
Add some cmdlets to interact with jmx

### LINKED ISSUE #50: JNet shall introduce a PowerShell module
**Is your feature request related to a problem? Please describe.**
PowerShell is a great tool and seems there is no integration with Java.

**Describe the solution you'd like**
Create a module/snapin, based on JNet, to interact with Java from PoweShell and publish it to [PowerShell Gallery](https://www.powershellgallery.com/). Maybe its name is **JNetPS**.

See https://learn.microsoft.com/en-us/powershell/scripting/dev-cross-plat/writing-portable-modules?view=powershell-7.2 for further information.

The list of CmdLet to be created will be defined later.

**Describe alternatives you've considered**
N/A

**Additional context**
See masesgroup/JNet#49

### LINKED ISSUE #110: Add some cmdlets to interact with JMX
Seems JNetPS does not contain any cmdlet (see #50) to interact with JMX

_Originally posted by @mariomastrodicasa in https://github.com/masesgroup/JNet/issues/104#issuecomment-1411182279_

### PR DESCRIPTION
**Pull request by masesdevelopers**
## Description

The PR adds many cmdlet to manage JMX from PowerShell. Meanwhile updates JNetPS and JNetTemplates.

## Related Issue

Closed #110 
Fix #50 

## How Has This Been Tested?

Test case from https://docs.oracle.com/javase/tutorial/jmx/remote/custom.html

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [x] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

**Comment by masesdevelopers** [comment]
Waiting for a review from @mariomastrodicasa

**Review by mariomastrodicasa — APPROVED** [review]

**Comment by mariomastrodicasa** [comment]
@masesdevelopers maybe it is useful to add cmdlet related to the types implementing https://github.com/masesgroup/JNet/blob/master/src/net/JNet/JavaX/Management/NotificationFilter.cs like reported in https://docs.oracle.com/javase/8/docs/api/javax/management/NotificationFilter.html. Plus add info related to the cmdlet in https://github.com/masesgroup/JNet/blob/5c842614c1bf653adfef43567e6338cf096f2769/src/net/Documentation/articles/usagePS.md

**Pull request merged**
